warehouse/static/sass/warehouse.scss

```diff
@@ -344,3 +344,8 @@ time {
   clip: rect(0, 0, 0, 0);
   border: 0;
 }
+
+// Class for unstyled lists
+.no-bullets {
+  list-style-type: none;
+}
```

Comment:
Would `unstyled` work instead of adding a new class?
https://github.com/pypi/warehouse/blob/097fb304b9b9653210a4501c806f6eb88f59df31/warehouse/static/sass/warehouse.scss#L143-L147

Reply:
Unfortunately it doesn't look right because it lacks padding/margins.

Reply:
Fair enough. In that case, should it be moved to either https://github.com/pypi/warehouse/blob/main/warehouse/static/sass/base/_lists.scss or should it be located in a block?